packages/api/src/period/utils.ts

```diff
@@ -53,6 +55,27 @@ const receiversWithScores = async (
   return receiversWithQuantificationScores;
 };
 
+const quantifiersWithCounts = (
+  quantifiers: PeriodDetailsQuantifier[]
+): PeriodDetailsQuantifierDto[] => {
+  const quantifiersWithQuantificationCounts = quantifiers.map((q) => {
+    let finishedCount = 0;
+    q.quantifications.forEach((quantification) => {
+      finishedCount = isQuantificationCompleted(quantification)
```

Comment:
What do you think about refactoring this `forEach` to be a `filter`, and then just count the number of quantifications? with `.length` -- I think it's more readable to avoid for loops when possible.

Reply:
actually i first did it that way and than changed to this because it should be faster but i think it will not impact user experience either way... ok, will revert my changes